Source/AlgorithmCYK.ParallelCPU/AlgorithmCYK.ParallelCPU.fs

```diff
@@ -0,0 +1,20 @@
+﻿module AlgorithmCYK.ParallelCPU
+open AlgorithmCYK
+open System.Threading.Tasks
+
+let matrixCYKParallelCPU (rules : list<string * string>) (nonterm : string array) (str : string) (n : int) (matrix : int array [][]) = 
+    Parallel.For(0, n, fun k -> allRulesCheck rules nonterm matrix.[n - 1].[k] [|(str.[k].ToString())|]) |> ignore

```

Comment:
Уф. Я рановато коммент повесил. А в чём проблема-то?

Regards,
Semyon Grigorev.

2016-05-30 16:51 GMT+03:00 SusaninaJulia <notifications@github.com>:

> In Source/AlgorithmCYK.ParallelCPU/AlgorithmCYK.ParallelCPU.fs
> <https://github.com/YaccConstructor/Brahma.FSharp/pull/36#discussion_r65074576>
> :
>
> > @@ -0,0 +1,20 @@
> > +﻿module AlgorithmCYK.ParallelCPU
> > +open AlgorithmCYK
> > +open System.Threading.Tasks
> > +
> > +let matrixCYKParallelCPU (rules : list<string * string>) (nonterm : string array) (str : string) (n : int) (matrix : int array [][]) =
> > +    Parallel.For(0, n, fun k -> allRulesCheck rules nonterm matrix.[n - 1].[k] [|(str.[k].ToString())|]) |> ignore
>
> Да, действительно, здесь заменила на iteri, а вот как это сделать во
> втором случае не знаю
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/YaccConstructor/Brahma.FSharp/pull/36/files/3d7e97288c62b86b19c5af9dae2b7ca7e1cb4595#r65074576>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGhQBA_7b8wI5G9n1ERnFfQTiiebPs3ks5qGutNgaJpZM4IpoRO>
> .
>